docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,93 @@
+
+The following snippet lies in the heart of the “run.sh” script:
+
+```
+if [[ ! ${SKIP_MINIKUBE_START} ]]; then
+    bash ${CURRENT_DIR}/../scripts/minikube.sh --domain "${DOMAIN}" --vm-driver "${VM_DRIVER}"
+fi
+
+bash ${CURRENT_DIR}/../scripts/build-kyma-installer.sh --vm-driver "${VM_DRIVER}"
+
+bash ${CURRENT_DIR}/../scripts/generate-local-config.sh
+
+if [ -z "$CR_PATH" ]; then
+
+    TMPDIR=`mktemp -d "${CURRENT_DIR}/../../temp-XXXXXXXXXX"`
+    CR_PATH="${TMPDIR}/installer-cr-local.yaml"
+
+    bash ${CURRENT_DIR}/../scripts/create-cr.sh --output "${CR_PATH}" --domain "${DOMAIN}"
+    bash ${CURRENT_DIR}/../scripts/installer.sh --local --cr "${CR_PATH}"
+
+    rm -rf $TMPDIR
+else
+    bash ${CURRENT_DIR}/../scripts/installer.sh --cr "${CR_PATH}"
+fi
+```
+
+The subsequent paragraphs describe respective sub-scripts triggered during the installation process.
+
+## The minikube.sh script
+
+> **NOTE:** To work with Kyma, use only the provided scripts and commands. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you don't need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
+
+Purpose of `installation/scripts/minikube.sh` is to configure and start Minikube. The script also checks if your development environment is cofigured to handle Kyma installation. This includes checking Minikube and Kubectl versions. If Minikube is already initialized, the system prompts you to agree to remove the previous Minikube cluster. The script exits if you don't want to restart your cluster.
+
+Minikube is configured to disable default nginx ingress controller.
+
+>**NOTE:** For the complete list of parameters passed to the `minikube start` command, refer to the `installation/scripts/minikube.sh` script.
+
+Once Minikube is up and running, the script adds local installation entries to `/etc/hosts`.
+
+## The build-kyma-installer.sh script
+
+Installer is application based on a [Kubernetes operator](https://coreos.com/operators/). Its purpose is to install Helm charts defined in the `installation` Custom Resource. Kyma-Installer is a Docker image that bundles Installer binary with Kyma charts. 
+
+The `installation/scripts/build-kyma-installer.sh` script extracts Kyma-Installer image name from the `installer.yaml` deployment file and uses it to build a Docker image inside Minikube. This image contains local Kyma sources from `resources` folder. 
+
+>**NOTE:** For Kyma-Installer Docker image details, refer to the `kyma-installer/kyma.Dockerfile` file.
+
+## The generate-local-config.sh script
+
+The Azure-Broker sub-component is a part of the `core` deployment that provisions managed services in the Microsoft Azure cloud. To enable Azure-Broker, export the following environment variables:
+ - AZURE_BROKER_SUBSCRIPTION_ID
+ - AZURE_BROKER_TENANT_ID
+ - AZURE_BROKER_CLIENT_ID
+ - AZURE_BROKER_CLIENT_SECRET
+
```

Comment:
...and that is done irrespective of the `run.sh` script / as part of it / additionally?